my_game.py

```diff
@@ -178,36 +184,47 @@ def on_draw(self):
 
         # Draw players score on screen
         arcade.draw_text(
-            "SCORE: {}".format(self.player_score),  # Text to show
-            10,                  # X position
-            SCREEN_HEIGHT - 20,  # Y positon
-            arcade.color.WHITE   # Color of text
+            "SCORE: {}".format(self.opposite_angle),  # Text to show
+            10,  # X position
+            SCREEN_HEIGHT - 20,  # Y_position
+            arcade.color.WHITE  # Color of text
         )
 
     def on_update(self, delta_time):
         """
         Movement and game logic
         """
 
-        # Calculate player speed based on the keys pressed
-        self.player_sprite.change_x = 0
-
-        # Move player with keyboard
-        if self.left_pressed and not self.right_pressed:
-            self.player_sprite.change_x = -PLAYER_SPEED_X
-        elif self.right_pressed and not self.left_pressed:
-            self.player_sprite.change_x = PLAYER_SPEED_X
-
         # Move player with joystick if present
         if self.joystick:
-            self.player_sprite.change_x = round(self.joystick.x) * PLAYER_SPEED_X
+            self.player_sprite.change_x = round(self.joystick.x) * 4
 
         # Update player sprite
         self.player_sprite.update()
 
         # Update the player shots
         self.player_shot_list.update()
 
+        if self.left_pressed:
+            self.player_sprite.angle += PLAYER_SPEED
+        else:
+            self.player_sprite.angle = self.player_sprite.angle
+
+        if self.right_pressed:
+            self.player_sprite.angle += -PLAYER_SPEED
+        else:
+            self.player_sprite.angle = self.player_sprite.angle
+
+        if not self.up_pressed:
+            if not self.player_sprite.speed < 0.05:
+                if self.down_pressed or self.right_pressed or self.left_pressed:
+                    self.player_sprite.speed -= 0.1
+                else:
+                    self.player_sprite.speed -= 0.001
+        else:
+            if not self.player_sprite.speed > 6:
+                self.player_sprite.speed += 0.05
```

Comment:
We don't want magic numbers here. Please use variables. I'm pretty sure you added those values to the Player class in lines 50 to 52.